apis/scheduling/v1alpha1/device_types.go

```diff
@@ -27,6 +27,7 @@ const (
 	GPU  DeviceType = "gpu"
 	FPGA DeviceType = "fpga"
 	RDMA DeviceType = "rdma"
+	XPU  DeviceType = "xpu"
```

Comment:
Should we reuse the GPU here? Adding a new device type may require all dependent parties to make changes. But I see that this PR actually does not have any changes to these dependent parties.

Reply:
+1, we should definitely reuse the GPU type